content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -84,7 +84,7 @@ to verify that backend Pods are working OK, so that kube-proxy in iptables mode
 only sees backends that test out as healthy. Doing this means you avoid
 having traffic sent via kube-proxy to a Pod that's known to have failed.
 
-{{< figure src="/images/docs/services-iptables-overview.svg" title="Services overview diagram for iptables proxy" class="diagram-medium" >}}
+{{< figure src="/images/docs/services-iptables-overview-1.svg" title="Services overview diagram for iptables proxy" class="diagram-medium" >}}
```

Comment:
@sftim 
kept the old image as it in the image path, created new image with little name difference [services-iptables-overview **-1** .svg], that 'sway [virtual-ips.md] page also updated with new image name.

Reply:
But why keep the old image? I don't see that we need to. (Git does that for us regardless).

Reply:
do i have to rebase again

Reply:
No need to rebase. We (maintainers) can squash commits down if that turns out to be useful.

Reply:
Thank you

Reply:
A second thought on this. Maybe a replacement is better than adding a new page.
Otherwise, the old images must be kept there because they are referenced by localized pages.

Reply:
If a localized page started to use the new image, why would that be bad?